src/ar_miner_log.erl

```diff
@@ -46,7 +46,8 @@ watchdog() ->
 	after ?FOREIGN_BLOCK_ALERT_TIME ->
 		log(
 			"WARNING: No foreign blocks received from network. "
-			"Please check your internet connection."
+			"Please check your internet connection and port forwarding settings."
+			"Consult your ISP or cloud provider to forward incoming HTTP requests with port 1984 to your machine."
```

Comment:
Hardcoding 1984 is not correct. The real values is in meta_db with the key `port`. E.g. `ar_meta_db:get(port)`.